Update amp.hbs

simarpreetsingh-019 opened the pull request:
amp-youtube script added, as according to  issue freeCodeCamp/freeCodeCamp#39075 .  

Suggestion: 
I was thinking of idea to add features like autopause, that will come to play If the user has interacted with the YouTube video (e.g., mutes/unmutes, pauses/resumes, etc.), and the video is scrolled in or out of view, the state of the video remains as how the user left it. , where should i add its box? I got a bit confused that's why I didn't added it.

ahmadabdolsaheb commented:
to test the discourse integration in rocks, add the following snippet.
            <div id='discourse-comments' class="post-full-footer"></div>

            <script type="text/javascript">
                var absoluteUrl = '{{url absolute="true"}}'.replace(".org/", ".rocks/");
                DiscourseEmbed = {
                    discourseUrl: 'https://www.freecodecamp.rocks/forum/',
                    discourseEmbedUrl: absoluteUrl
                };
                (function () {
                    var d = document.createElement('script'); d.type = 'text/javascript'; d.async = true;
                    d.src = DiscourseEmbed.discourseUrl + 'javascripts/embed.js';
                    (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(d);
                })();
            </script>


for production, use the following snippet
            <div id='discourse-comments'></div>

            <script type="text/javascript">
                DiscourseEmbed = {
                    discourseUrl: 'http://www.freecodecamp.org/forum/',
                    discourseEmbedUrl: '{{url absolute="true"}}'
                };
                (function () {
                    var d = document.createElement('script'); d.type = 'text/javascript'; d.async = true;
                    d.src = DiscourseEmbed.discourseUrl + 'javascripts/embed.js';
                    (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(d);
                })();
            </script>

simarpreetsingh-019 committed:
Update amp.hbs

amp-youtube script added, as according to  issue #39075 .  
If i want to add features like autopause, that will come to play If the user has interacted with the YouTube video (e.g., mutes/unmutes, pauses/resumes, etc.), and the video is scrolled in or out of view, the state of the video remains as how the user left it. , where should i add its box? i got a bit confused that's why I didn't added it.

ahmaxed commented:
We can discuss that functionality in a separate issue and see the community's feedback.

Thanks for submitting this PR.
I tried this solution locally by embedding a youtube video in a post, updating it and visiting the /amp url, but was not able to see the youtube video.

Could you please let me know if you have tested your solution locally and share the steps to test your solution if possible.
Thank you.

simarpreetsingh-019 commented:
so actually there is no such additional thing to insert hyperlinks in amp pages.

The basic < a href = ""> command is used here also to insert links, but the youtube script is necessary if we want to show a video in a specific dedicated block made for showing  or adding a box for showing youtube or vimeo video(vimeo script needed). verify [here](https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#ancr)

I searched in repo's file, but youtube script was not present in any of them, so it is added.
Additionally to use it , we have to add specific block to show it. 

The feature suggestion was made accordingly for that block.
